src/submenu/submenu.sss

```diff
@@ -8,13 +8,27 @@ a.submenu_link
   display: block
   height: 33px
   padding: 0 9px
-  overflow: hidden
+  overflow-x: hidden
+  overflow-y: visible
```

Comment:
Why do you want to remove hiding scroll until hover?

Reply:
It needs for expand the active zone by y-axis on ```<a>``` element